src/schedule-and-details/requirements-section/grade-requirements/index.jsx

```diff
@@ -52,7 +52,7 @@ GradeRequirements.propTypes = {
   errorEffort: PropTypes.string,
   entranceExamMinimumScorePct: PropTypes.string,
   onChange: PropTypes.func.isRequired,
-  isEditable: PropTypes.bool,
+  isEditable: PropTypes.bool.isRequired,
```

Comment:
You can remove `isEditable` from the `defaultProps` section if it is now required.